source/unified-test-format/unified-test-format.rst

```diff
@@ -724,12 +720,30 @@ See the `Versioned API <../versioned-api/versioned-api.rst>`__ spec for more
 details on these fields.
 
 
-collectionData
-~~~~~~~~~~~~~~
+initialCollectionData
+~~~~~~~~~~~~~~~~~~~~~
 
-List of documents corresponding to the contents of a collection. This structure
-is used by both `initialData`_ and `test.outcome <test_outcome_>`_, which insert
-and read documents, respectively.
+List of documents corresponding to the contents of a collection. This
+structure is used by `initialData`_.
+
+The structure of this object is as follows:
+
+- ``collectionName``: Required string. See `commonOptions_collectionName`_.
+
+- ``databaseName``: Required string. See `commonOptions_databaseName`_.
+
+- ``collectionOptions``: Optional object. Options that MUST be passed to the
+  ``create`` command when creating the collection.
```

Comment:
Can we link to https://docs.mongodb.com/manual/reference/command/create/ here? Since we don't have a collection management spec (as we do with indexes), that's going to be the primary resource for anyone trying to determine what options are supported.

I would also add a line that tests SHOULD NOT utilize the `writeConcern` option, since that could conflict with the "majority" write concern we require later on (feel free to note that as the justification).

Reply:
Added the link and a note about `writeConcern`.